README.md

```diff
@@ -42,8 +42,7 @@ client.cluster.templates
 
 #### Index
 
-The methods in the [Index](lib/elastomer/client/index.rb) class deal with the
-management of indexes in the cluster. This includes setting up type mappings
+Index has methods to manage clusters.This includes setting up type mappings
```

Comment:
Thank you for your contribution to the project :) Could you please elaborate on why you feel this change improves the documentation? I think it's useful to maintain a link to the class being described. You will also need to fix the space that was removed between the two sentences.